test/lightstep-opentelemetry-launcher-node.test.ts

```diff
@@ -127,6 +133,78 @@ describe('Lightstep OpenTelemetry Launcher Node', () => {
       });
     });
 
+    describe('hostname', () => {
+      const stubbedHostname = 'hostymcstubs.local';
```

Comment:
If you use sinon sandbox in beforeEach (`sandbox = sinon.createSandbox();`) then you can simply call 
`sanbox.restore();` in afterEach

and for example in line 189 instead of hardcoded value you can also use mock
```
// process.env.HOSTNAME = 'envhost.local'
// do instead
sandbox.stub(process.env, 'HOSTNAME').value('envhost.local')
```
which will be mocked then automatically and will never interfere with any other tests etc. or when this value has been already set - you are removing `HOSTNAME`  in line 198

Reply:
I updated to use the sandbox. I wasn't able to do:

```
sandbox.stub(process.env, 'HOSTNAME').value('envhost.local')
```

as it resulted in `TypeError: Cannot stub non-existent property HOSTNAME`.

This did work though:

```
sandbox.stub(process, 'env').value({ HOSTNAME: 'envhost.local' });
```

Reply:
hmm I saw it somewhere very recently and was almost sure it was like this but it seems I'm wrong though, anyway I'm glad it works with `{}`.